worker.go

```
@@ -240,6 +247,10 @@ func (w *worker[T, R]) processNextJob() {
 	w.CurProcessing.Add(1)
 	j.ChangeStatus(processing)
 
+	if q, ok := w.Queue.(IAcknowledgeable); ok {
+		q.PrepareForFutureAck(j.ID(), v)
```

golangci-lint (1.64.8)

Error return value of `q.PrepareForFutureAck` is not checked

(errcheck)